website/docs/r/no_code_module.html.markdown

```diff
@@ -9,9 +9,6 @@ description: |-
 
 Creates, updates and destroys no-code module for registry modules.
 
-**NOTE:** This resource is currently in beta and isn't generally
-available to all users. It is subject to change or be removed.
-
```

Comment:
Should the skipUnlessBeta(t) be removed, too? Has the feature flag been removed?

Reply:
No. The feature flag will not be removed until when the feature is poised to be released.

Reply:
Let's keep this note until the next release, just in case.

Reply:
Can we merge this documentation fix without cutting a new release?

Reply:
No, we have to cut a patch release

Reply:
Keeping the note until the next release. I'd be glad if we can cut a patch release to fix the docs. Thanks